code/index.html

```diff
@@ -3,28 +3,84 @@
 
-    <!-- Signup form -->
+    <header>
+      <video autoplay loop muted preload="auto">
+        <source src="./yogadesktop.mp4" type="video/mp4">
+        Your browser does not support video.
+      </video>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <img class="img-tablet" src="./yogatablet.jpeg" alt="Yoga picture">
+      <img class="img-phone" src="./yogaphone.jpeg" alt="Yoga picture">
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <h1>the Studio</h1>
+    </header>
+
+    <main class="content-container">
+      <div class="content-item">
+        <form action="http://httpbin.org/anything" method="POST">
+          <h2>Sign up!</h2>
+
+          <label for="name">Name</label><br>
+          <input type="text" id="name" name="name" autocomplete="off"> 
+          <label for="e-mail">E-mail</label><br>
+          <input type="text" id="e-mail" name="e-mail" autocomplete="off"><br>
+      
+          <p>Choose your city</p>
+          <input type="radio" id="Stockholm" name="city" value="Stockholm" checked="checked">
+          <label for="Stockholm">Stockholm</label><br>
+          <input type="radio" id="Berlin" name="city" value="Berlin">
+          <label for="Berlin">Berlin</label><br>
+
+          <div class="checkbox">
+            <input type="checkbox" id="newsletter" name="newsletter" value="yes" checked="checked">
+            <label for="newsletter">I want to recieve the newsletter</label><br>
+            <input type="checkbox" id="terms" name="terms" value="yes">
+            <label for="terms">I agree to the <span>terms & conditions</span></label><br>
+          </div>
+
+          <input class="submit" type="submit" value="Submit">
+        </form>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="content-item">
+        <section class="accordion-container">
+          <h2>FAQ</h2>
+
+          <div class="accordion-item">
+            <div class="question" id="question1">How do I become a member?</div>
+            <div class="answer">
+              <p>Signup by filling out the form on this site.</p>
+            </div>
+          </div>
+          <div class="accordion-item">
+            <div class="question" id="question2">What does it cost?</div>
+            <div class="answer">
+              <p>500 sek/50 euro per month.</p>
+            </div>
+          </div>
+          <div class="accordion-item">
+            <div class="question" id="question3">How do I end my membership?</div>
+            <div class="answer">
+              <p>Send an e-mail to thestudio@gmail.com.</p>
+            </div>
+          </div>
+        </section>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    </main>
+
+  <footer>
+    
+  </footer>
```

Comment:
Remove unused code!

Reply:
It is styled though, so not completely unused!